WordPress/WordPress-Alpha.entitlements

```diff
@@ -6,9 +6,6 @@
 	<array>
 		<string>webcredentials:wordpress.com</string>
 		<string>webcredentials:*.wordpress.com</string>
-		<string>applinks:wordpress.com</string>
```

Comment:
Is the goal is to disable only the Reader links?

Reply:
What would this change disable?

Reply:
The plan at the moment (see the Slack conversation linked in the PR description) is
1. WordPress should not handle WP.com universal links. Jetpack should be the only app for that.
2. We'll keep all the universal links as they are for now.

If we'd like to remove some links, we can do that from the server side. The WordPress app changes in this PR needs to go out in the next release.

Reply:
The app no longer has:

- Notifications
- Reader
- Stats

So I'm comfortable disabling all WP.com links – we can let the web app handle those if needed.